content/en/apps/guides/messaging/rapidpro.md

```diff
@@ -0,0 +1,114 @@
+---
+title: RapidPro SMS Gateway
+linkTitle: "RapidPro"
+weight:
+description: >
+    Integration for sending and receiving SMS
+relatedContent: >
+    apps/reference/app-settings/sms
+    apps/guides/forms/app-form-sms
+    apps/guides/messaging/sms-states
+    apps/guides/messaging/message-loops
+    apps/guides/messaging/shortcodes
+---
+
+
+As of v3.11.0, SMS messages can be sent and received using [RapidPro](https://community.rapidpro.io/).
+
+## RapidPro configuration
+
+### Store globals
+
+Generate a long unique key to use as the `medic-api-key`.
```

Comment:
Is there a reason we call this `medic-api-key` instead of `cht-api-key`?

Reply:
Either one works, the name is irrelevant. just a convention to be used in this doc. We can also call it "bubblegum_jewel" :D

Reply:
The sole reason for `medic-api-key` is that it's the same name used in the Africa's Talking documentation (and it was my 
"model")

Reply:
Got it, thanks. If it's just a couple minutes to change in both places, it's probably worth it.  If it takes us down a deep rabbit hole, probably not worth it right now.

Reply:
Thanks for the switch!